mono/mini/TestDriver.cs

```diff
@@ -13,10 +13,40 @@ public class CategoryAttribute : Attribute
 		get; set;
 	}
 }
+public class TestDriverReporter
+{
+	private int failedTests = 0;
+	private int skippedTests = 0;
+	private int executedTests = 0;
+
+	public int FailedTests {
+		get {
+			return failedTests;
+		}
+	}
+
+	public int SkippedTests {
+		get {
+			return skippedTests;
+		}
+	}
+
+	public int ExecutedTests {
+		get {
+			return executedTests;
+		}
+	}
+
+	public void ReportResults (int executed, int skipped, int failed) {
```

Comment:
I thought about it but since the purpose of the class is simple and won't affect behavior of caller it felt like it would only introduce unnecessary overhead.